### PR TITLE
Updating curator-operator fetchdata and report crds

### PR DESCRIPTION
**Pull request by skanthed**
/assign @dystewart

### CODE REVIEW

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/fetchdata.curatoroperator.io/customresourcedefinition.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
```
Comment (HumairAK) [inline cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/fetchdata.curatoroperator.io/customresourcedefinition.yaml:6]: this value is not needed, and will end up failing our upcoming [kubeval](https://github.com/instrumenta/kubeval) ci check

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
```
Comment (HumairAK) [inline cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml:7]: same as above

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.curatoroperator.io/customresourcedefinition.yaml [hunk 3]
```diff
@@ -4,6 +4,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
```
Comment (HumairAK) [inline cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.curatoroperator.io/customresourcedefinition.yaml:7]: same as above

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/fetchdata.curatoroperator.io/customresourcedefinition.yaml (hunk 1 above)
Comment (skanthed) [reply cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/fetchdata.curatoroperator.io/customresourcedefinition.yaml:6]: Sure. Will remove and push the changes.

**Comment by skanthed** [comment]
@HumairAK updated with the changes. But just wanted to know why [kubeval](https://github.com/instrumenta/kubeval) will fail ci check for null timestamp?

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+                  - pod_usage_memory_byte_seconds
+                  - pod_request_memory_byte_seconds
+                  - node_capacity_cpu_cores
+                  - node_capacity_cpu_core_seconds
+                  - node_capacity_memory_bytes
+                  - node_capacity_memory_byte_seconds
+                  - pod_limit_memory_byte_seconds
+                  - node_capacity_cpu_cores
+                  type: string
+                maxItems: 11
```
Comment (Gregory-Pereira) [inline cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml:54]: there are 12 elements here, shouldnt that be the max item value?

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+                minItems: 1
+                type: array
+              namespace:
+                minLength: 0
+                type: string
+              reportPeriod:
+                description: 'Specifies how to treat concurrent executions of a Job.
```
Comment (Gregory-Pereira) [inline cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml:61]: If you wish to make the description multiline and to show values with -, put 
```
description: |
    Specifies how to ... Valid values are:
        - Day
        - ...
```

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml (hunk 5 above)
Comment (skanthed) [reply cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml:61]: Yes, will take care next time.

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml (hunk 4 above)
Comment (skanthed) [reply cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml]: Updated.

File: cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml [hunk 6]
```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  name: reportapis.curator.operatefirst.io
```
Comment (Gregory-Pereira) [inline cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml:7]: The convention for naming of for CRDs is that the name of the custom resource definition should be the same as the name of the file.

Comment (skanthed) [reply cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml]: Updated for all the crds.

**Pull request merged**